.github/pull-request-template.md

```diff
@@ -4,9 +4,16 @@
 
 #### Description
 Describe your changes here.
+
```

Comment:
Also worth adding an exception for docs changes.

Reply:
I think we're going to have a docs updates section in the release tags too. Might be worth keeping it.

Reply:
Release notes for docs? :/

Reply:
FIIIINNNNNNNEEEEEEEEEEEEEEE